bentoml/_internal/frameworks/xgboost.py

```diff
@@ -31,6 +29,7 @@
     )
 
 MODULE_NAME = "bentoml.xgboost"
+MODEL_FILENAME = "model.ubj"
```

Comment:
I think we should probably do this instead of a global `MODEL_NAMESPACE`; we should never change this and I think it improves code readability to not combine two constants from another file here.

Reply:
I agree, let's remove the global model namespace